fenix/app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -413,7 +414,15 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
                 pref.title = getString(R.string.preferences_sync_now)
                 pref.isEnabled = true
 
-                val time = getLastSynced(requireContext())
+                val lastSyncedTime = getLastSynced(requireContext())
```

Comment:
nit: we could move this into a single func as it's doing the same thing in both cases?